docs/tutorials/jokes.md

```diff
@@ -2994,7 +2994,13 @@ export async function createUserSession(
 
 <summary>app/routes/login.tsx</summary>
 
-```tsx filename=app/routes/login.tsx lines=[18] nocopy
+```tsx filename=app/routes/login.tsx lines=[4,18] nocopy
+// ...
+
+import { db } from "~/utils/db.server";
+import { login, createUserSession } from "~/utils/session.server";
+import stylesUrl from "~/styles/login.css";
```

Comment:
I think we can hide the css import here, it's not used/relevant for the example

Reply:
Good point.